lms/static/js/spec/edxnotes/views/tabs/tags_spec.js

```diff
@@ -0,0 +1,80 @@
+define([
+    'jquery', 'underscore', 'js/common_helpers/template_helpers', 'js/spec/edxnotes/helpers',
+    'js/edxnotes/collections/notes', 'js/edxnotes/collections/tabs',
+    'js/edxnotes/views/tabs/tags', 'js/spec/edxnotes/custom_matchers',
+    'jasmine-jquery'
+], function(
+    $, _, TemplateHelpers, Helpers, NotesCollection, TabsCollection, TagsView,
+    customMatchers
+) {
+    'use strict';
+    describe('EdxNotes TagsView', function() {
```

Comment:
No, it hasn't been rolled out yet. @mikigoyal mentioned it to both me and Andy, and as a result I've been trying to be more "strategic" about where I am adding tests.